_posts/2016-10-27-journal-of-a-junior-multisite.md

```diff
@@ -0,0 +1,139 @@
+---
+layout:         post
+title:          "Journal of a Junior: Using one Umbraco instance to host multiple sites"
+date:           2016-10-27
+author_id:      "maggy_burns"
+description:    "Hello, Maggy here! Here's my second blog post in my Journal of a Junior Series. A tutorial on how to host multiple sites on one Umbraco instance."
+categories:     space between
+banner:         "/assets/images/blog/tutoring.jpeg"
+---
+
+Hello! Maggy here on my long overdue second blog post in my Journal of a Junior series! This is my sixth month working at Space Between, I have worked on so many projects, with so many amazing technologies it was hard to pick what to write this post on. I’ve decided to go with using one Umbraco CMS to host multiple sites!
+
+
+I've chosen this because it was one of the tasks I thought was going to be very difficult. It involved so many things I'd never encountered before, and was a big learning curve - but very rewarding and now something that I realise can be done in a flash! This tutorial will assume you are familiar with the Umbraco CMS and being able set up different domain names in IIS for each site.
+
+
+###Step 1: Setting up
```

Comment:
I would suggest starting at Creating DocTypes and just mention in it that people can see the documentation page for getting started.

This section doesn't really say anything about getting started.